examples/main/main.cpp

```diff
@@ -721,7 +719,6 @@ int main(int argc, char ** argv) {
 
                     is_interacting = true;
                     printf("\n");
-                    console::set_display(console::user_input);
                     fflush(stdout);
```

Comment:
Should we move this `fflush(stdout)` after `console::set_display`?

Reply:
Not only does set_display flush its output stream after it prints, but printf will too if it contains a newline and stdout is a console, so no fflush should be needed.

Reply:
I removed the `fflush()` call.